### ISSUE TITLE
Problem with Rollup

### ISSUE THREAD

**Issue by Josema** [vinzdef/three-svg-loader#1]
```
index.js:13 Uncaught TypeError: Cannot add property SVGLoader, object is not extensible
    at index.js:13
    at createCommonjsModule (three.module.js:47311)
    at three.module.js:47311
    at index.js:198
```

Here the line:
```js
THREE.SVGLoader = function ( manager ) {
```

**Comment by vinzdef** [comment]
Hi @Josema, is this an issue you have at build or runtime? 
If runtime, can you provide a minimal set for reproducing it?
I just built a rollup bundle without errors.

**Comment by Josema** [comment]
Maybe I'm using it in a wrong way. Can you provide an example of how to use it with THREE.js?

The docs does not have any example at all.

**Comment by vinzdef** [comment]
Hi @Josema.
I am sorry I didn't understand well your issue.

It is now fixed in **v0.2.4**!

[Here](https://github.com/bugged-reality/three-svg-loader-rollup-example) you can find a working example.
